raycicmd/make.go

```diff
@@ -61,6 +62,20 @@ func parsePipelineFile(file string) (*pipelineGroup, error) {
 	return g, nil
 }
 
+func getTagFilter(config *config) ([]string, error) {
+	tagFilterScript := config.TagFilterScript
+	if tagFilterScript == "" {
+		return nil, nil
+	}
+	cmd := exec.Command("bash", "-c", tagFilterScript)
+	filters, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("tag filter script: %w", err)
+	}
+
+	return strings.Fields(string(filters)), nil
```

Comment:
maybe we should just remove all the `=1` suffix? or add a flag into the python script so that it does not generate the `=1` but just the env var names?

Reply:
right I think that script itself should removes the =1 suffix, raycicmd shouldn't do that; the other script is still used by civ1 and requires the =1 though. We can remove the =1 completely from that script once civ1 is done with.

Reply:
https://github.com/ray-project/ray/pull/39837 added an option to remove the =1 flag from the determine_tests_to_run.py script